apps/interpolate_sample.py

```diff
@@ -50,6 +55,11 @@ def parse_args():
         choices=['lerp', 'slerp'],
         default='lerp',
         help='mode to sample from endpoints\'s interpolation.')
+    parser.add_argument(
+        '--proj-latent',
+        type=str,
+        default=None,
+        help='Projection file of Images. Produced by stylegan_projector.py.')
```

Comment:
Projection image files. 
You may specify the content in this file. After reading the help information, I cannot know what you want to achieve with this argument.